exonum-java-binding/core/rust/src/runtime/paths.rs

```diff
@@ -28,7 +28,7 @@ pub fn system_classpath() -> String {
     let mut jars = Vec::new();
     let jars_directory = {
         let mut executable_directory = executable_directory();
-        executable_directory.push("lib/java");
+        executable_directory.push("../lib/java");
```

Comment:
Do scripts launching an app with `cargo run` work with such change? If I understand correctly, the app Java dependencies are always copied to `lib/java` somewhere alongside the application.